docs/user_guide.md

```diff
@@ -73,70 +74,96 @@ All logs can be listed by clicking "Logs". You can apply a filter by username, o
+## Searching projects and repositories  
+Entering a keyword in the search field at the top lists all matching projects and repositories. The search result includes both public and private repositories you have access to.  
 
 ![browse project](img/new_search.png)
 
-##Administrator options
-###Managing user
-Administrator can add "administrator" role to an ordinary user by toggling the switch under "Administrator". To delete a user, click on the recycle bin icon.  
+## Administrator options  
+### Managing user  
+Administrator can add "Administrator" role to an ordinary user by click button on the left and select "set as Administrator". To delete a user, select "Delete". 
 
 ![browse project](img/new_set_admin_remove_user.png)
 
-###Managing destination
-You can list, add, edit and delete destinations in the "Destination" tab. Only destinations which are not referenced by any policies can be edited.  
+### Managing endpoint  
+You can list, add, edit and delete endpoints in the "Endpoints" tab. Only endpoints which are not referenced by any enabled rules can be edited.  
 
-![browse project](img/new_manage_destination.png)
+![browse project](img/new_manage_endpoint.png)
 
-###Managing replication
-You can list, edit, enable and disable policies in the "Replication" tab. Make sure the policy is disabled before you edit it.  
+### Managing replication  
+You can list, edit, enable and disable rules in the "Replication" tab. Make sure the policy is disabled before you edit it.  
 
 ![browse project](img/new_manage_replication.png)
 
-##Pulling and pushing images using Docker client
+### Managing authentication
+You can change authentication mode between DB(by default) and LDAP before any user is added,when there is at least one user(besides admin) in Harbor,you cannot change the authentication mode.  
+![browse project](img/new_auth.png)
+When use LDAP mode, self-registration is disabled.The parameters with stars are essential, for more info, refer to [User account](#user-account).   
+![browse project](img/ldap_auth.png)
+
+### Managing project creation
+You can manage if a non-admin user can create a project.  
+![browse project](img/new_proj_create.png)
+
+### Managing self registration
+You can manage if an unregistered user can sign up for an account. This option is not availiable if you use LDAP authentication.  
+![browse project](img/new_self_reg.png)
+
+### Managing verify remote cert
+You  can choose to whether to verify remote endpoint's certification. You must disable certificate verification if the remote registry uses a self-signed or an untrusted certificate.  
+![browse project](img/new_remote_cert.png)
+
+### Managing email settings
+You can change Harbor's email settings, the mail server is used to send responses to users who request to reset their password.  

```

Comment:
You can change Harbor's email settings, the mail server is used to send out responses to users who request to reset their password.